core/shared/src/test/scala/bobcats/HotpSuite.scala

```diff
@@ -16,32 +16,23 @@
 
 package bobcats
 
-import cats.Functor
 import cats.effect.IO
-import cats.syntax.functor._
-import munit.CatsEffectSuite
 import scodec.bits._
 
-import scala.reflect.ClassTag
-
-class HotpSuite extends CatsEffectSuite {
+class HotpSuite extends CryptoSuite {
 
   val key = hex"3132333435363738393031323334353637383930"
 
   val expectedValues = List(
     755224, 287082, 359152, 969429, 338314, 254676, 287922, 162583, 399871, 520489
   )
 
-  def tests[F[_]: Hmac: Functor](implicit ct: ClassTag[F[Nothing]]) = {
```

Comment:
This move from tagless final is not voluntary; because the fixture isn't pure, we run into issues with instantiation.

For now, I'll switch to a concrete IO, and maybe look at `weaver` if it works better with `cats-effect`.